lstchain/tools/lstchain_create_calibration_file.py

```diff
@@ -2,103 +2,77 @@
 Extract flat field coefficients from flasher data files.
 """
 import numpy as np
-from traitlets import Dict, List, Unicode, Float, Bool
-
-
-from ctapipe.core import Provenance, traits
-from ctapipe.io import HDF5TableWriter
-from ctapipe.core import Tool
-from ctapipe.io import EventSource
 from ctapipe.containers import PixelStatusContainer
+from ctapipe.core import Provenance, Tool, traits
+from ctapipe.io import EventSource, HDF5TableWriter
+
 from lstchain.calib.camera.calibration_calculator import CalibrationCalculator
 from lstchain.calib.camera.r0 import CameraR0Calibrator
 from lstchain.io.lstcontainers import LSTEventType
-
-__all__ = [
-    'CalibrationHDF5Writer'
-]
+from tqdm.autonotebook import tqdm
 
 
 class CalibrationHDF5Writer(Tool):
-    """
-     Tool that generates a HDF5 file with camera calibration coefficients.
-     This is just an example on how the monitoring containers can be filled using
-     the calibration Components in calib/camera.
-     This example is based on an input file with pedestal and flat-field events
-
-     For getting help run:
-     python calc_camera_calibration.py --help
-
-     """
 
     name = "CalibrationHDF5Writer"
     description = "Generate a HDF5 file with camera calibration coefficients"
 
+    one_event = traits.Bool(False, help="Stop after first calibration event").tag(config=True)
 
-    one_event = Bool(
-        False,
-        help='Stop after first calibration event'
+    output = traits.Path(
+        help="Name of the output file",
+        directory_ok=False,
+        default_value="calibration.hdf5"
     ).tag(config=True)
 
-    output_file = Unicode(
-        'calibration.hdf5',
-        help='Name of the output file'
-    ).tag(config=True)
-
-    log_file = Unicode(
-        'None',
-        help='Name of the log file'
```

Comment:
Hi,
This log_file trailet was originated by the wish to give the name of the file where the log output should be written, unfortunately at that time I did not menage to use it successfully (we use now a unix pipe in the calibration script), is there some news on this possibility?

Reply:
It is builtin in ctapipe 0.10

Reply:
Perfect. Let's remember it for a next version then, for the moment the trailet is indeed not necessary.